README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
+  destinationHoldDuration: 3000
+})
+
+const json = packet.deserializeIlqpLiquidityRequest(binary)
+```
+
+#### QuoteLiquidityResponse
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityResponse({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  liquidityCurve: Buffer.alloc(16), // Must be a buffer of size (n * 16) bytes
```

Comment:
> I'm not a fan if it means the library getting that buffer needs to have an OER parser.

It doesn't need an OER parser. The format of a liquidity curve is an array of UInt64s, it does not use any OER-specific types.

> How would you use the UInt32Array? Is the idea that that would work as long as none of the numbers exceed the precision of the UInt32Array?

You'd map the buffer to a UInt32Array. Since the buffer contains a bunch of uint64s, each cell would be the high, low of each of the numbers. For example, to get the coordinates for the n-th point, you would do:

``` js
// create a liquidity curve with a single point <0,0>
const curveBuffer = Buffer.alloc(16, 0)

// convert Node buffer to Uint32Array
const curve = new Uint32Array(curveBuffer.buffer)

// get the n-th point (counting from 0)
const n = 0

const x_hi = curve[n * 4]
const x_lo = curve[n * 4 + 1]
const y_hi = curve[n * 4 + 2]
const y_lo = curve[n * 4 + 3]
```

The fastest way to continue from here is to do Math using the hi, lo numbers directly. (You'd use the same algorithms we used to use to do 64-bit on 32-bit processors.) Or you can convert to a BigNumber like so: 

``` js
const HIGH_WORD_MULTIPLIER = 0x100000000
const x = new BigNumber(x_hi).times(HIGH_WORD_MULTIPLIER).add(x_lo)
```